next.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('next').NextConfig} */
+const path = require('path')
+
+module.exports = {
+  reactStrictMode: true,
+  sassOptions: {
+    includePaths: [path.join(__dirname, 'styles')],
```

Comment:
Is it necessary to include that path in order do load the sass files properly? Doesn't next handle this OotB?

Reply:
In case you wanted to expose all styles globally, check out _app: https://nextjs.org/docs/advanced-features/custom-app

Reply:
Oh I corrected it. I copied that code from the official doc on using sass